controllers/ProposalController.php

```diff
@@ -676,4 +747,109 @@ private function saveEditedComment(Comment $comment, $newCommentContent)
             throw new CannotSaveException($comment);
         }
     }
+
+    /**
+     * Display proposals for a Publisher
+     * or single proposal
+     *
+     * @param null $id
+     * @return string
+     */
+    public function actionManageProposals($id = null)
+    {
+        if (!is_null($id)) {
+            $selectedProposal = $this->checkIfProposalExists($id);
+            $displayItems = $this->buildOneProposalDisplayItems($selectedProposal);
+
+            return $this->render('proposal', $displayItems);
+        }
+
+        $approvedProposalsQuery = $this->buildApprovedProposalsQuery();
+        $approvedProposals = $this->buildApprovedProposalsActiveDataProvider($approvedProposalsQuery);
+        $notApprovedProposals = $this->getNotApprovedProposals($approvedProposalsQuery);
```

Comment:
The functions names 'buildApprovedProposalsActiveDataProvider' and 'getNotApprovedProposals' do the same thing (returns an ActiveDataProvider), so one of their names should be changed to be consistent

Reply:
Yes you're right but getNotApprovedProposals is the most consistent name, if you compare it with methods which displays an Active data Provider for reviewers. 
So we have two options. 
First we rename all functions that returns an active data provider like buildXXXXActiveDataProvider(). Second, we assume that this case is special and we change anything.